config/default.js

```diff
@@ -421,4 +423,9 @@ module.exports = {
     // e.g., 20210531_amo_download_funnel_experiment: true,
     '20210714_amo_vpn_promo': true,
   },
+
+  addonIdsWithQRCodes: [
```

Comment:
I think (and I am not saying we should add that here) that we need to verify this list of IDs against the list of IDs retrieved in the `bin/fenix-qr-codes.js` script. If we can load the config in this script, that could be as easy as:

```js
const config = require('config');

// Somewhere after `Promise.all()` I suppose...

const knownIds = new Set(config.get('addonIdsWithQRCodes'));
const addonIdsFromAPI = [...new Set(addons.map(({ id }) => id))];

if (addonIdsFromAPI.filter(x => knownIds.has(x)).length > 0) {
  console.log(`'addonIdsWithQRCodes' might be outdated:`);
  console.log(`  addonIdsWithQRCodes=${JSON.stringify(knownIds.sort())}`);
  console.log(`  IDs from API=${JSON.stringify(addonIdsFromAPI)}`);
  process.exit(1);
}
```

Reply:
I had to do some fiddling to deal with the `Cannot use import statement outside a module` error I was getting, but I was able to make it work. There seemed to be some kind of logic problem with your `filter`, and in trying to make it work I just removed all the `Set`s and just used arrays and it seems fine. Even if we end up with duplicate id's (which we shouldn't) it won't make any difference. I've added a new commit with my version of this change.